scripts/download-env.sh

```diff
@@ -31,6 +31,20 @@ if [ ! -z "$SETTINGS_FILE" -o ! -z "$XILINX" ]; then
 	exit 1
 fi
 
+# Conda does not support ' ' in the path (it bails early).
+if echo "${SETUP_DIR}" | grep -q ' '; then
```

Comment:
Can you indent this to match the existing code?

Reply:
Fixed.